corehq/apps/userreports/tests/utils.py

```diff
@@ -31,9 +32,9 @@ def get_sample_report_config():
 
 def get_sample_data_source():
     folder = os.path.join(os.path.dirname(__file__), 'data', 'configs')
-    sample_file = os.path.join(folder, 'sample_data_source.json')
+    sample_file = os.path.join(folder, 'sample_data_source.yaml')
```

Comment:
I think you may have forgotten to commit this file?

Reply:
actually that should have stayed json. fixed